changelog/4920.doc.rst

```diff
@@ -0,0 +1,2 @@
+Added a gallery example (:ref:`sphx_glr_generated_gallery_plotting_zoom_map_example.py`)
+showing how to add a zoomed-in region of an image.
```

Comment:
```suggestion
Added a gallery example (:ref:`sphx_glr_generated_gallery_plotting_zoom_map_example.py`) demonstrating how to add a zoomed-in region next to a full disk image.
```

Reply:
Do we have to make any changes Nabil! Right now i need to clean up my git and local repo..i will look into it how to do that. 
also please have a look once which one is better. let me know if i have to update it! will do it! [Link](https://github.com/sunpy/sunpy/pull/4920#discussion_r575649043)

Reply:
I am not sure I follow? I think these changes (among others) will have to be made before we can merge this.

Reply:
I can see the changes. I am sorry i fall asleep last night! thanks :smiling_face_with_three_hearts:

Reply:
Please don't apologize for sleeping.